src/recon_test/test_blocks_on_cylindrical_projectors.cxx

```diff
@@ -53,6 +53,7 @@
 #include "stir/recon_buildblock/BackProjectorByBinUsingProjMatrixByBin.h"
 #include "stir/IO/write_to_file.h"
 #include "stir/VoxelsOnCartesianGrid.h"
+#include "stir/PixelsOnCartesianGrid.h"
```

Comment:
needed? I'd rather get rid of this clas in the future

Reply:
It is needed for `centre_axial_values[z] = volume_sptr->get_plane(z).at(0).at(0);`. Is there another way to achieve this?

Reply:
what about `(*volume_sptr)[z][0][0]`, or can use `at` of course. or `(*volume_sptr)[make_coordinate(z,0,0)]`.